.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: test
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - '2.6'
+          - '2.7'
+          - '3.0'
+          - '3.1'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+      - uses: actions/cache@v2
```

Comment:
Why not use `bundler-cache: true` ? It also runs `bundle install` automatically.
https://github.com/ruby/setup-ruby/#caching-bundle-install-automatically

Reply:
Ah, I haven't used that feature for some reason. Good to know. Fixed https://github.com/cookpad/expeditor/pull/47/commits/b326d0815aad0cda3c6cbd316a1e49360463750d